lib/new_relic/telemetry/broadway.ex

```diff
@@ -0,0 +1,216 @@
+defmodule NewRelic.Telemetry.Broadway do
+  @moduledoc """
```

Comment:
@binaryseed I've followed documentation for Plug instrumentation, actually even copied a bulk of it! Add some description and examples how this will map on NewRelic.

I'm not a native speaker, so extra dose of review here is more than welcome 🤓